yt/chyt/controller/internal/auth/ythttputil.go

```diff
@@ -63,6 +63,14 @@ func GetCredentials(r *http.Request) (yt.Credentials, error) {
 		return credentials, nil
 	}
 
+	if cookie, err := r.Cookie("access_token"); err == nil {
```

Comment:
Is this a request from ui? In that case cookie called `yt_oauth_access_token` and it is passed to proxies as `access_token `

Reply:
@Flunt1k said that it is cookie from ui backend, not browser, so should be ok when UI will send cookie instead of oauth token to the proxie